### ISSUE TITLE
Add functionality to give actual host and port for Docker articles.

### ISSUE THREAD

**Issue by liamawhite** [WASdev/sample.ferret#3]
Where the actual host and port are the hostname and port of the Liberty instance actually processing the request.

The functionality is needed to demonstrate the load balancing capabilities of both Liberty/IHS and Docker in a container based deployment. I am happy to write the functionality but would like some feedback on the best place to add it.

The most sensible place I can think of is as follows

```
...
remoteHost	ingress-endpoint.ingress
remotePort	60622
remoteUser	null
localHost	7c3b9bd26d08
localPort	9080
...
```

**Comment by liamawhite** [comment]
PR -> https://github.com/WASdev/sample.ferret/pull/4

**Comment by jgawor** [comment]
Merged #4. Thanks!
